fcomm/src/bin/fcomm.rs

```diff
@@ -356,6 +388,35 @@ impl Verify {
     }
 }
 
+impl Put {
+    async fn put(&self) -> Result<(), Error> {
+        let store = &mut Store::<Scalar>::default();
+
+        let src = read_from_path(store, &self.data)?;
+        store.hydrate_scalar_cache();
+        let (scalar_store, _) = ScalarStore::new_with_expr(store, &src);
+        let ipld = to_ipld(scalar_store.clone()).unwrap();
+        let cid = dag_put(&self.host, ipld)
+            .await
+            .expect("Failed to store on IPFS");
+        println!("{:?}\nstored on IPFS", cid);
+        Ok(())
+    }
```

Comment:
Ideally, these (put and get) would use `expression(…)` and support the option to either read/write Lurk syntax or the underlying IPLD as elsewhere — since not all Lurk data has a  'source code syntax' representation.